src/test/java/teammates/test/cases/browsertests/AdminEmailPageUiTest.java

```diff
@@ -73,6 +73,49 @@ private void testCompose() throws Exception {
         assertTrue(isEmailComposeElementsPresent());
         emailPage.verifyStatus("Email will be sent within an hour to recipient@email.tmt");
 
+        ______TS("send email to group - invalid file type");
+
+        emailPage.clearRecipientBox();
+        emailPage.inputGroupRecipient("invalidGroupList.xlxs");
+        emailPage.verifyStatus("Group receiver list upload failed. Please try again.");
+
+        ______TS("send email to group - no subject");
+
+        emailPage.inputContent("Email Content");
+        emailPage.clearRecipientBox();
+        emailPage.clearSubjectBox();
+        emailPage.inputGroupRecipient("validGroupList.txt");
+        emailPage.verifyGroupListFileKey(emailPage.getGroupListFileKey());
+        emailPage.verifyStatus("Group receiver list successfully uploaded to Google Cloud Storage");
+        emailPage.clickSendButton();
+        assertTrue(hasStatusMessageNoSubject());
+
+        ______TS("send email to group - success");
+
+        emailPage.inputSubject("Email Subject");
```

Comment:
Here if you look the test case i .e. "send email - no subject" and "send email -success" same procedure has been used. 
Also, Isn't it fine if we are just adding an email subject?